python/pyspark/shell.py

```diff
@@ -30,6 +30,7 @@
 
 import pyspark
 from pyspark.core.context import SparkContext
+from pyspark.logger import SPARK_LOG_SCHEMA
```

Comment:
Where is SPARK_LOG_SCHEMA used in this file? What am I missing?

Reply:
it is import in the pyspark shell so that users can use the constant directly.